pkg/rngd/cmd/rngd/main.go

```diff
@@ -14,7 +15,76 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+type rng struct {
+	rngName  string
+	rngDev   string
+	disabled bool
```

Comment:
Should this not be `enabled`, so the default is false, and then you can enable it when an `init` succeeds?

Reply:
Yes. enabled==true means a valid entropy source which can be used.

Reply:
Got it. Just a nitpick, but it is the reverse now, because the default for `disabled` is `false`

Reply:
Ah, the code will set `disabled` as `true` for an invalid/malfunction entropy source, so when try to write the entropy to the pool, we have below code:
```
if e.disabled {
     continue
}
```
My comments is confusing 😄

Reply:
As I said, "just a nitpick". :-)